### DISCUSSION TITLE
solara_enterprise Auth login/out doesn't point to the right URL

### DISCUSSION THREAD

**Discussion by francesfeng** [widgetti/solara#436] [Q&A]
Hi all, 

I have the following code that used to sign in/out users, from this doc https://solara.dev/api/avatar

```
if not auth.user.value:
   solara.Button("Register/Login", style=Login_menu_css, href=auth.get_login_url())
else:
   with auth.AvatarMenu():
       solara.Button("Logout", icon_name="mdi-logout", href=auth.get_logout_url(), text=True)
```

The app is deployed on Heroku and have DNS points to a custom domain hosted on GoDaddy. However when I login/out from the app hosted on the custom domain, it always return to the URL assigned by Heroku afterwards. 

Is the description clear, happy to provide more details?

Many thanks!

**Comment by maartenbreddels** [comment]
Hi Francis,

Usually, solara knows what the 'base url' is (e.g. the address typed in the browser location basically), but it seems we fail to get the correct one.
You can override this using the following env var, e.g.:
SOLARA_BASE_URL=https://solara.dev

This is now documented in d2615d5931e15ec40e6ba3ba87b72e3e1b3ff1cb
